### DISCUSSION TITLE
AAC encodes getting .aac filename extension

### DISCUSSION THREAD

**Discussion by strange-antiquark** [enzo1982/freac#343] [Help & Support] [answered]
I've been converting some MP3 files to AAC. When I first started doing this, the output files got a .m4a extension.
Now, however, the extension has become .aac, and there seems to be nothing I can do to change this back. The encoder configuration is set to AAC, version MPEG-4, extension .m4a and has not been changed since I began using freac.

This is a great utility, btw. I just wish I had discovered it years ago.

**Comment by enzo1982** [comment, marked as answer]
You need to set the file format to MP4 in the format options to create .m4a files (after setting the file format to MP4, you will be able to choose the file extension between .m4a, .m4b, .m4r and .mp4).

The AAC file format option will create raw AAC files with an .aac extension.

**Comment by strange-antiquark** [comment]
Well, duh. I must have changed that and forgotten about it -- and more recently, I didn't think to test to see if there was any difference between the two format settings. I feel so embarrassed, to put it mildly!!

TYVM.

**Reply by enzo1982** [reply]
No worries, that can happen. Glad I was able to help.
